packages/daimo-api/src/api/profile.ts

```diff
@@ -134,6 +135,10 @@ export class ProfileCache {
     return this.linkedAccounts.get(addr) || [];
   }
 
+  getProfilePicture(addr: Address) {
+    return `${daimoDomainAddress}/profile/${addr}`;
```

Comment:
nit but i recommend making this `${daimoDomainAddress}/profile/${addr}/pfp.png`

...so that we can put a more complete profile at `/profile/<addr>` in future

Reply:
Good point. I'll make the change.